javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -2186,7 +2186,7 @@ class WebElement {
     if (!this.driver_.fileDetector_) {
       return this.schedule_(
           new command.Command(command.Name.SEND_KEYS_TO_ELEMENT).
-              setParameter('text', keys).
+              setParameter('text', keys.then(keys => keys.join(''))).
```

Comment:
@jleyba - well, so this is where the code is a bit weird (and hard to follow). I wanted to keep the change as minimal as possible.

That whole "second block" starting at 2198 deals with passing in a file path (for file inputs) I believe.  Anyways, it generates the `keys` value, and on line 2201 calls `keys.join()`.  So at that point, I believe `keys` is a string (not an array).

So line 2205: `text` is string
And line 2206: `value` is an array

Which, after my change, *should* match.   In the existing code:
line 2189: `text` is an array
line 2190: `value` is an array

What I am trying to do with my change is make:
line 2189: `text` is a string
line 2190: `value` is an array

That way `text` is always a string, and `value` is always an array.

With that said, this is my first change to this repo, and I don't want to break things for people.  I did testing locally with the latest FF/Chrome geckodriver/chromedriver, and things worked.  I also ran the unit tests (which *seemed* to work for my changes).  I think they are in a bad state currently, b/c a few tests failed before and after my changes.